### ISSUE TITLE
stopPropagation on actions is broken

### ISSUE THREAD

**Issue by bazubii** [adamghill/django-unicorn#515]
It looks like at some point, the action event handlers were moved to the document (checking that the target is correct). It seems like this totally breaks the .stop modifier. That calls stopPropagation, but that doesn't work if its an event on the document. There is nothing to stop propagation to. The intended behavior would seem to be to stop propagation from the element the action is on to parent elements, but that isn't working if the event is actually on the document.

**Comment by adamghill** [comment]
They got moved to the document because the events were getting re-added on every update. That would cause the event to fire multiple times. I'm pretty sure I tried a few approaches to prevent it from happening, but resorted to keeping the events on the document as a last resort. If there is a smarter approach I'd love to check it out.

**Comment by bazubii** [comment]
Ah that makes sense. I don't have a proposed solution as of yet, but the docs at least are incorrect in saying it'll work.  I'll have to think and play with it a bit.

**Comment by adamghill** [comment]
> but the docs at least are incorrect in saying it'll work

True. If there isn't an easy solution I could remove it from the docs (at least for now).
